WordPress/UITestsFoundation/Screens/StatsScreen.swift

```diff
@@ -16,6 +23,38 @@ public class StatsScreen: ScreenObject {
         )
     }
 
+    public func verifyStatsAreLoaded(_ stats: [String]) -> Bool {
```

Comment:
Oops, I did it again. ®️ 
Would it be the same for assertion methods like `assertStatsAreLoaded()`?  🤔

Reply:
> Would it be the same for assertion methods like assertStatsAreLoaded()?

I tend to say yes, because from a code style guide POV (which doesn't really cover this point), it should be the same for all cases.